torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1,4 +1,5 @@
 
+#include <torch/csrc/distributed/c10d/Work.hpp>
```

Comment:
nit: is this needed? as ProcessGroupNCCL.h somehow already has access to Work?

Reply:
This was suggested by `visual studio`.  I removed it!